### ISSUE TITLE
istioctl analyze <dir> should analyze all files in provided directory

### ISSUE THREAD

**Issue by selmanj** [istio/istio#20437]
**Describe the feature request**

It's a bit annoying to type `find my-dir -type f | xargs istioctl analyze`. It would be ideal if I could just provide a directory rather than a file as an argument. Instead we get the error:
```
$ istioctl analyze .
Error(s) adding files: 1 error occurred:
        * read .: is a directory
```

**Describe alternatives you've considered**
None.

**Additional context**

**Comment by esnible** [comment]
@selmanj What should the Milestone be for this?

**Comment by selmanj** [comment]
This is done - I forgot I had filed an issue for this.

This was implemented in #20718 which was cherry-picked to 1.5
